dvc/remote/ssh/connection.py

```diff
@@ -106,6 +106,9 @@ def makedirs(self, path):
                 # Since paramiko errors are very vague we need to recheck
                 # whether it's because path already exists or something else
                 if e.errno == errno.EACCES or not self.exists(path):
+                    logger.exception(
```

Comment:
We shouldn't print it here, we just need to wrap it something like
```
raise DvcException("unable to create remote directory '{}'".format(path), cause=exc)
```
Also please note '' around {}. We are using it in other places, as it helps to distinguish part of the message that was auto-generated from the one that was pre-set. Sometimes helps to debug weird things 🙂

Reply:
Got it, @efiop !